otx/algorithms/action/configs/detection/base/ava.py

```diff
@@ -0,0 +1,89 @@
+"""Data configuration for AVA dataset."""
```

Comment:
The name `ava_dataset.py` seems more appropriate.

Reply:
Actually ava_dataset.py is named from mmaction. As mmaction only supports AVA dataset, so they name dataset code for action detection as ava_dataset.py. It is needed to change ava_dataset.py to detection_dataset.py. 
This config is for AVA dataset. It shares dataset class with other dataset format, however some config info is different.
Therefore the description looks right

Reply:
You mean the file name must be different from `ava_dataset.py`? If so, I agree with this name. Or `ava_data_pipeline.py` seems fine too. It looks good as it is.

Reply:
ava_data_pipeline.py looks great thanks